INSTALL.rst

```diff
@@ -13,19 +53,58 @@ Create a virtual environment and clone the INSPIRE source code using `git`:
     $ workon inspirehep
     (inspirehep)$ cdvirtualenv
     (inspirehep)$ mkdir src
-    (inspirehep)$ git clone https://github.com/inspirehep/inspire-next.git inspirehep
+    (inspirehep)$ git clone https://github.com/inspirehep/inspire-next.git src/inspirehep
 
 
 Install requirements
 ====================
 
-Use `pip` to install all requirements:
+Use `pip` to install all requirements, it's recommended to upgrade pip and
+setuptools to latest too:
 
 .. code-block:: console
 
+    (inspirehep)$ pip install --upgrade pip setuptools
     (inspirehep)$ cdvirtualenv src/inspirehep
     (inspirehep)$ pip install -r requirements.txt --pre --exists-action i
 
+And for development:
+
+.. code-block:: console
+
+    (inspirehep)$ pip install -e .[development]
+
+
+Custom configuration and debug mode
+====================================
```

Comment:
Note also that commit 8dacc07d7 introduced another way of overriding configuration via a file called `inspirehep/config_local.py`. I'm not sure it's worth mentioning in the docs, I believe only @eamonnmag uses this variant.